src/dealDashboard/dealDiscussion/discussionsList/discussionsList.scss

```diff
@@ -101,3 +116,47 @@
 
 }
 
+.mobile.only {
+  display: none;
+}
+.desktop.only {
+  display: inline;
+}
```

Comment:
Can these be scoped and not global?
Maybe useful, but for now I'd prefer scoped